packages/core/src/index.ts

```diff
@@ -60,17 +65,25 @@ export type AbbyConfig<
   flags?: Flags;
   settings?: Settings<F.NoInfer<FlagName>, Flags>;
   debug?: boolean;
+  flagCacheConfig?: FlagCacheConfig;
 };
 
 export class Abby<
   FlagName extends string,
   TestName extends string,
   Tests extends Record<string, ABConfig>,
-  Flags extends Record<FlagName, FlagValueString>
+  Flags extends Record<FlagName, FlagValueString>,
 > {
   private log = (...args: any[]) =>
     this.config.debug ? console.log(`core.Abby`, ...args) : () => {};
 
+  private testDevtoolOverrides: Map<keyof Tests, Tests[keyof Tests]["variants"][number]> =
+    new Map();
+
+  private flagDevtoolOverrides: Map<FlagName, boolean> = new Map();
```

Comment:
guess this shouldn't be "just booleans" anymore, right?
the fix should be another PR :)